packages/core/test/unit/patchers/aws3_p.test.js

```diff
@@ -99,14 +124,39 @@ describe('AWS v3 patcher', function() {
         }
       })();
 
+      ddbGetRequest = new (class PutCommand {
```

Comment:
Is this meant to mock a specific DynamoDB `Put<>` command?
The only Put<> type command whitelisted is `PutItem`, so maybe this can mock the `PutItemCommand` specifically, and use the [whitelisted attributes](https://github.com/aws/aws-xray-sdk-node/blob/e32dd9580def3556ddee50b0090e2c9b97bcf361/packages/core/lib/resources/aws_whitelist.json#L78-L86)

Reply:
Yeah it was supposed to be a GetItemCommand. I was trying with PutItem before I realized it wasn't whitelisted, and forgot to update it here! I'll change it to be more accurate to what's being tested